apps/delivery-options/src/views/MyParcelDeliveryOptions/Pickup/PickupLocationList/PickupLocationCarrierGroup.vue

```diff
@@ -78,8 +78,8 @@ const {
     return locationCode.value === option.value;
   },
 });
-
+const carrierIdentifier = location.value?.carrier;
```

Comment:
dit is niet meer reactive en wordt nu alleen bij mount van het component geresolved naar een waarde. werkt het niet als je `location.value?.carrier` binnen die computed aanroept?

Reply:
Ik heb het aangepast. Ik heb de identifier gebruikt die in de locatie staat. Ik denk dat dit prima is want eerder (in useOptionsGroupedByCarrier.ts regel 27) wordt deze zelfde waarde ook gebruikt om de naam op te zoeken.

Reply:
maar nu pak je de eerste optie en niet de gekozen carrier, hoe kan dat werken?